.envrc

```diff
@@ -103,3 +103,5 @@ fi
 export FISSILE_AUTH_TYPE=rbac
 export FISSILE_OUTPUT_DIR="${FISSILE_OUTPUT_DIR:-${PWD}/output/helm}"
 export FISSILE_DEFAULTS_FILE=
+
+export INGRESS_CONTROLLER=${INGRESS_CONTROLLER:-}
```

Comment:
So this just sets it to empty if unset?
Seems unnecessary to put it in `.envrc` (since it has the side effect of resetting your value if you:

1. `cd ~/scf`
2. `set INGRESS_CONTROLLER=nginx` # forgot to do this
3. `do stuff …`
4. `cd ..` # for whatever reason
5. `cd scf` # oops is now unset

I believe step 4 could even be `cd src/uaa-fissile-release` because it reloads the `.envrc`.

Reply:
True. I initially thought it wouldn't hurt, but I guess you convinced me to take it out of here.